pkg/apis/core/validation/validation.go

```diff
@@ -75,6 +75,8 @@ var iscsiInitiatorIqnRegex = regexp.MustCompile(`iqn\.\d{4}-\d{2}\.([[:alnum:]-.
 var iscsiInitiatorEuiRegex = regexp.MustCompile(`^eui.[[:alnum:]]{16}$`)
 var iscsiInitiatorNaaRegex = regexp.MustCompile(`^naa.[[:alnum:]]{32}$`)
 
+var filePathRegex = regexp.MustCompile(`^(?:/[^/]+)+$`)
```

Comment:
exclude some special characters, try `^(/[^/?!; ]*)+/?$` and add a test case e.g. `/etc/ceph/keyring;/etc/foo/bar`